enhancements/sdk-external-and-pluggable-validations.md

```diff
@@ -0,0 +1,494 @@
+* as a validation author, I can write a validation for a bundle and use it
+  without having to get a new operator-sdk release
+
+TODO: Stories from the epic; consolidate with the above stories
+
+#### Story 3
+* Users want to edit/add/remove a set of validation rules for Operator bundle validation.
+
+#### Story 4
+* Users can host a set of validation rules either locally or in remote.
+
+#### Story 5
+* Users can define the target set of validation rules for SDK's bundle validation command (either from local or remote source).
+
+### Implementation Details/Notes/Constraints [optional]
+
+There are a few alternatives, but I think wrapping the validations in their
+own executables makes sense. It aligns with the work we've done with
+[Phase 2][phase-2] plugins. It allows the most flexibility in terms of
+implementation. It also has a simple API -
+input: bundle dir; output: ManifestErrors JSON.
+
+Wrapping validations in their own executable simply means that there
+needs to be something that the operator-sdk can run like a shell script
+or a binary.
+
+The validation executable will need to accept a single input value which
+is the bundle directory. The output will need to be a ManifestErrors JSON. This
+JSON will be parsed by the operator-sdk converted and output as a Result.
+
+For example, the existing validations could easily be wrapped with a `main.go`
+file and compiled into a binary. They could be copied to their own repos for
+easier release. Now you might be thinking that this means we'll have many
+releases still, we actually would only have one (1) release which is the
+validator itself. For go based validators this might mean a new binary, but you
+could also release your validator in python and make it a single file or even a
+shell script.
+
+From the operator-sdk's point of view, we don't care what you use
+to create your validator only that we can run it and pass it a bundle directory.
+
+
+TODO: how will validators be found?
+
+#### Validators
+
+##### Running validators
+
+The [existing validators](#default-validators-run-by-operator-sdk) operate
+on a bundle. Each validator should be some executable that accepts a
+single argument, the bundle root directory. So `operator-sdk` will pass them
+the bundle directory. It will be the validator's responsibility to parse
+the bundle to get what it needs.
+
+For example, the validator executable will be invoked from the
+`operator-sdk` in the following manner:
+
+```
+/path/to/validator/executable /path/to/bundle
+```
+
+A concrete example might look like this:
+
+```
+./validator-poc /home/user/dev/gatekeeper-operator/bundle
+```
+
+The actual implementation is up to the author. Here we have an example of
+the [`OperatorHubValidator`][validator-poc1] as a Go binary.
+
+##### Validator results
+
+As stated earlier, each validator should be some executable that accepts a
+single argument, the bundle root directory. Th validators should also output
+ManifestErrors JSON to stdout.
```

Comment:
```suggestion
ManifestResult JSON to stdout.
```
should not it be ManifestResult?

Reply:
Yes, sorry about that.  I will update the EP.  https://github.com/operator-framework/api/blob/master/pkg/validation/errors/error.go#L9-L16